composer.json

```
@@ -12,18 +12,19 @@
     "require": {
         "php": ">=8.1",
         "php-64bit": "*",
-        "symfony/config": "^5.4 || ^6.0 || ^7.0",
-        "symfony/dependency-injection": "^5.4 || ^6.0 || ^7.0",
-        "symfony/filesystem": "^5.4 || ^6.0 || ^7.0",
-        "symfony/polyfill-mbstring": "^1.19"
+        "symfony/config": "^5.4 || ^6.0 || ^7.0 || ^8.0",
+        "symfony/dependency-injection": "^5.4 || ^6.0 || ^7.0 || ^8.0",
+        "symfony/filesystem": "^5.4 || ^6.0 || ^7.0 || ^8.0",
+        "symfony/polyfill-mbstring": "^1.19",
+        "symfony/yaml": "^5.4 || ^6.0 || ^7.0 || ^8.0"
     },
     "require-dev": {
         "brianium/paratest": "^7.3",
         "easy-doc/easy-doc": "0.0.0 || ^1.2.3",
         "friendsofphp/php-cs-fixer": "^3.57",
         "gregwar/rst": "^1.0.6",
-        "phpstan/phpstan": "2.1.31",
-        "phpunit/phpunit": "^10.5.20,<10.5.32",
+        "phpstan/phpstan": "~2.1.32",
+        "phpunit/phpunit": "^10.5.20,<10.5.32|^11.0.0",
```

Just for consistency with the rest of the file:
```suggestion
        "phpunit/phpunit": "^10.5.20,<10.5.32 || ^11.0.0",
```